README.md

```diff
@@ -32,6 +32,12 @@ var ReactCrop = require('react-image-crop');
 import ReactCrop from 'react-image-crop';
 ```
 
+If you are using TypeScript, install also `@types/react-image-crop` and then import (types will get picked up):
+
+```ts
```

Comment:
Do you need to import with `* as default`? If so it might be related to this issue which I plan to fix: https://github.com/DominicTobias/react-image-crop/issues/178

Reply:
Ad of now, yes, and that issue does seem related. There is also something like [`--allowSyntheticDefaultImports ](https://www.typescriptlang.org/docs/handbook/compiler-options.html).

You can verify this quickly by using `create-react-app test --scripts-version=react-scripts-ts`.